test/react_lazy_test.dart

```diff
@@ -0,0 +1,26 @@
+@TestOn('browser')
+library react.react_lazy_test;
+
+import 'package:react/react.dart' as react;
+import 'package:react/react_client/react_interop.dart';
+import 'package:test/test.dart';
+
+import 'factory/common_factory_tests.dart';
+
+main() {
+  group('lazy', () {
```

Comment:
Oh, one more piece of test coverage: could we test that if the future returned by lazy component errors, that that case is handled (to verify the Dart Future error rejects the JS promise correctly)?

When that happens, React should throw an error in the component that propagates to the error boundary.

react-dart doesn't have a canonical ErrorBoundary utility component like over_react does, but there's [this one](https://github.com/Workiva/react-dart/blob/91c9c2cf1eb1bc224f7078d2b6ca06cce8b2f259/test/js_builds/shared_tests.dart#L205-L223) that could be copied over or split out into a shared test utility file.

Reply:
done